timed/employment/views.py

```diff
@@ -53,30 +53,31 @@ class UserViewSet(ModelViewSet):
 
     def get_queryset(self):
         user = self.request.user
-        current_employment = models.Employment.objects.get_at(
-            user=user, date=datetime.date.today()
-        )
         queryset = get_user_model().objects.prefetch_related(
             "employments", "supervisees", "supervisors"
         )
 
-        if current_employment.is_external:
-            assigned_tasks = Task.objects.filter(
-                Q(task_assignees__user=user, task_assignees__is_reviewer=True)
-                | Q(
-                    project__project_assignees__user=user,
-                    project__project_assignees__is_reviewer=True,
+        if user.employments.all().exists():
+            current_employment = models.Employment.objects.get_at(
+                user=user, date=datetime.date.today()
+            )
+            if current_employment.is_external:
```

Comment:
```suggestion
            if not current_employment or current_employment.is_external:
```

Reply:
That doesn't really work, because I get an `DoesNotExist` error if there's no employment for the specific user at the current date.